logrotation-k8env/Dockerfile.cron

```diff
@@ -0,0 +1,17 @@
+FROM centos:7.9.2009
```

Comment:
Move all the K8s specific stuff into "logrotation/K8s". There should be no "K8s" specific keywords anywhere else outside of this folder.

Reply:
done